crates/router/src/types/domain.rs

```diff
@@ -54,8 +63,12 @@ pub use event::*;
 pub use merchant_connector_account::*;
 pub use merchant_key_store::*;
 pub use network_tokenization::*;
+pub use payment_method_data::*;
 pub use payment_methods::*;
+#[cfg(all(feature = "v2", feature = "payment_methods_v2"))]
 pub use payments::*;
```

Comment:
Why is the `payments` module exports behind the `payment_methods_v2` feature flag?